### PR TITLE
Demos: Added a new demo using Semantic UI

### PR DESCRIPTION
**Pull request by Arkni**

### CODE REVIEW

**Comment by staabm** [comment]
I guess the demo should mention in the body that it is using semantic ui and maybe even link to the semanticui project?

**Comment by Arkni** [comment]
I will do that.

**Comment by Arkni** [comment]
Fixed.
screenshot from 2015-09-26 16 32 21

File: demo/semantic-ui/index.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+	</style>
+</head>
+<body>
+    <div class="ui grid sem">
+        <div class="four wide column"></div>
+
+        <div class="eight wide column">
+			<div class="ui info message" style="top: 15px;">
+				<div class="header">
+					This demo is using the Semantic UI framework
```
Comment (Arkni) [inline demo/semantic-ui/index.html]: Should this note changed to: `This demo is created using Semantic UI framework` ?

Comment (staabm) [inline demo/semantic-ui/index.html]: Maybe " This demos shows How to integrate jquery-validation and the semantic ui framework"

Comment (Arkni) [inline demo/semantic-ui/index.html]: Sounds good to me, will fix it soon

Comment (Arkni) [inline demo/semantic-ui/index.html]: Fixed.
screenshot from 2015-09-27 12 48 31

**Comment by staabm** [comment]
thanks

**Pull request merged**